### PR TITLE
Adding a test of the change pre-save hook.

### PR DESCRIPTION
**Pull request by brandonsavage**
@peterbe r?

Looking for a better way to do the setup here, since repeating myself sucks.

### CODE REVIEW

File: dragnet/dll/tests.py [hunk 1]
```diff
@@ -38,3 +38,24 @@ def test_unauthenticated_user_redirected_from_edit(self):
...
+            first_name='System',
+        )
+
+        myfile = File.objects.create(
+            created_by=sys_user,
+            modified_by=sys_user,
+            file_name='abc.dll'
+        )
+
+        myfile.debug = '123'
```
Comment (peterbe) [inline dragnet/dll/tests.py:57]: What does this have to do with "updating_file_logs_updates"?
When you do the `File.objects.create(...)` it will fire a pre-save.

File: dragnet/dll/tests.py [hunk 2]
```diff
@@ -38,3 +38,24 @@ def test_unauthenticated_user_redirected_from_edit(self):
...
+        myfile.save()
+
+        changes = FileHistory.objects.filter(dll=myfile)
+        eq_(len(changes), 1)
+        change = list(changes).pop()
```
Comment (peterbe) [inline dragnet/dll/tests.py]: Use this instead

```
change, = changes
```

File: dragnet/dll/tests.py (hunk 1 above)
Comment (brandonsavage) [inline dragnet/dll/tests.py:57]: What I want to test is that when I make changes, it keeps a record of those changes. 

Naming FTL, but the above is the goal.

**Comment by peterbe** [comment]
r+ with nits

Question though: Why use a pre-save?! That gets fired even if the File save fails. 
pre-save is basically meant for preventing saves.
